packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/template.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import styled from 'styled-components'
+
+import { Button, Icon, Image } from 'blockchain-info-components'
+import {
+  HeaderTextWrapper,
+  IconBackground,
+  SceneHeader,
+  SceneHeaderText,
+  SceneSubHeaderText
+} from 'components/Layout'
+
+const Wrapper = styled.div`
```

Comment:
I'm seeing lot's of `styled` components in this file. I know this is how things are done now but I wonder if we could propose to start separating `styled-components` to another file like we used to (something like `template.style.tsx` perhaps) and export the `styled components` from there just in case we need to reuse and also to improve code readability.